test_configs/osx.exclude

```diff
@@ -2,3 +2,4 @@
 
 # Exclude msg_epoll test as OSX doesn't have epoll system call
 msg_epoll
+msg_sockets
```

Comment:
@vkrishna was this change intentional? My experiments on OSX do not fail this test when enabled. Or maybe it has been fixed since this commit?

Reply:
Yes. [https://github.com/ofiwg/libfabric/pull/4028](PR 4028) has fixed this.

Reply:
Submitted https://github.com/ofiwg/libfabric/pull/4825 to re-enable it. Thanks!